project/Dependencies.scala

```diff
@@ -42,27 +42,27 @@ object Dependencies {
   val sprayHttpx    = "io.spray"        %% "spray-httpx"   % Version.spray
   val sprayJson     = "io.spray"        %% "spray-json"    % Version.sprayJson
 
-  val apacheMath    = "org.apache.commons" % "commons-math3" % "3.5"
+  val apacheMath    = "org.apache.commons" % "commons-math3" % "3.6"
 
   val jettyWebapp   = "org.eclipse.jetty" % "jetty-webapp" % "8.1.0.RC4"
   val jerseyBundle  = "com.sun.jersey"    % "jersey-bundle" % "1.11"
-  val slf4jApi      = "org.slf4j"         % "slf4j-api" % "1.6.0"
+  val slf4jApi      = "org.slf4j"         % "slf4j-api" % "1.7.18"
   val asm           = "asm"               % "asm"       % "3.3.1"
-  
+
   val slick         = "com.typesafe.slick" %% "slick"      % "2.1.0"
-  val postgresql    = "postgresql"         % "postgresql"  % "9.1-901.jdbc4"
+  val postgresql    = "postgresql"         % "postgresql"  % "9.2-1002.jdbc4"
 
   val caliper       = ("com.google.code.caliper" % "caliper" % "1.0-SNAPSHOT"
     from "http://plastic-idolatry.com/jars/caliper-1.0-SNAPSHOT.jar")
 
-  val nscalaTime    = "com.github.nscala-time" %% "nscala-time" % "2.6.0"
+  val nscalaTime    = "com.github.nscala-time" %% "nscala-time" % "2.10.0"
 
-  val awsSdkS3      = "com.amazonaws" % "aws-java-sdk-s3" % "1.9.34"
+  val awsSdkS3      = "com.amazonaws" % "aws-java-sdk-s3" % "1.10.57"
```

Comment:
We should revert this dependency back. This has always been a tricky one with hadoop, and this version was meaningful, it was the one that let things work against `geotrellis-ec2-cluster`. Can you throw a comment in saying "Using this version of aws-java-sdk-s3 for compatibility reasons" in there?